src/variations/event-query/components/event-order-controls.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * WordPress dependencies
+ */
+import { SelectControl, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * EventOrderControls component
+ *
+ * @param {*} param0
+ * @return {Element} EventCountControls
+ */
+export const EventOrderControls = ({ attributes, setAttributes }) => {
+	const { query: { order, orderBy } = {} } = attributes;
+	const label =
+		order === 'asc'
+			? __('Ascending Order', 'gatherpress')
+			: __('Descending Order', 'gatherpress');
+	return (
+		<>
+			<SelectControl
+				label={__('Order Events By', 'gatherpress')}
+				value={orderBy}
+				help={
+					orderBy === 'meta_value' || orderBy === 'meta_value_num'
+						? __(
+								'Meta Value and Meta Value Num require that Meta Key is set in the Meta Query section.',
+								'gatherpress'
+							)
+						: ''
+				}
+				options={[
+					// The 'gatherpress_event' post_type does not support 'author'.
+					// {
+					// 	label: __( 'Author', 'gatherpress' ),
+					// 	value: 'author',
+					// },
```

Comment:
You can uncomment this, it does support Author now.

Reply:
Yes .... but, I can't imagine of any case, where ordering by author would be any usefull for events. If you can bring at least one scenario, for sure I could allow ordering by author here.

Reply:
> I could see a filter by author if 2 or more people are holding events for different groups of people on the same site.

An author filter is already available:

<img width="807" height="780" alt="image" src="https://github.com/user-attachments/assets/aa2c91f6-8957-472a-8a15-656060048d9e" />